_posts/2019-11-17-strimzi-at-kubecon-san-diego.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Strimzi at KubeCon+CloudNativeCon in San Diego"
+date: 2019-11-17
+author: strimzi_developers
+---
+
+From Tuesday, most of the Cloud Native and Kubernetes world meets in San Diego for the KubeCon+CloudNativeCon conference organized by Cloud Native Computing Foundation (CNCF). 
+As a CNCF Sandbox project, Strimzi will be of course represented there as well.
+Read more to learn how to meet Strimzi maintainers and see Strimzi demos.
+
+<!--more-->
+
+If you are attending KubeCon, you have two opportunities to see live demo of Strimzi in the mini-theather at Red Hat Booth (#D1):
+* On **Tuesday at 3:50pm**
+* On **Thursday at 11:10am**
+
+_(The full demo schedule is available [here](https://www.redhat.com/en/events/red-hat-kubecon-cloudnativecon-north-america-2019#demo-schedule))._
+
+After the demo, we will be available for any questions or discussions related to Strimzi and Apache Kafka.
+
+Of course, we know that KubeCon will be full of interesting talks.
+So if you cannot make it to any of the demos because of some other talk, don't worry.
+Jakub Scholz - one of our maintainers - will be available through the whole conference.
+So if you want to meet and talk about Strimzi or Aapche Kafka, just ping him on the #strimzi channel on [CNCF Slack](https://slack.cncf.io/) or at Twitter ([@strimziio](https://twitter.com/strimziio) or [@scholzj](https://twitter.com/scholzj).
```

Comment:
typo "Aapche"

Reply:
You are too slow as a reviewer ... fixed looooong time ago ;-)